src/integrator/LangevinThermostat.cpp

```diff
@@ -137,12 +140,34 @@ void LangevinThermostat::thermalizeAdr()
 
 void LangevinThermostat::frictionThermo(Particle& p)
 {
+    /*System& system = getSystemRef();*/
     real massf = sqrt(p.mass());
 
     // get a random value for each vector component
     Real3D ranval((*rng)() - 0.5, (*rng)() - 0.5, (*rng)() - 0.5);
 
+    // Test code to switch among different types of LGV thermostats
+    // mode(0): peculiar vel; mode(1): full vel (incl. shear speed);
+    // mode(2): ONLY thermalize along y-dir
+    // To activate custom modes, UNCOMMENT all "/* .. */"
+    /*int mode = system.lebcMode;
+
+    if (mode == 0)
```

Comment:
Can we introduce constants with meaningful names for these numbers?

Reply:
It's not for all users but I add more explanations there